assets/javascripts/initializers/communitarian.js.es6

```diff
@@ -90,6 +93,24 @@ function initializeCommunitarian(api) {
   });
 
   api.modifyClass("controller:create-account", {
+    onShow() {
+      this.setProperties({
+        accountUsername: Date.now()
```

Comment:
hm, is it a good? it's a temporatrily value for username validation skip?

Reply:
@Vadim-Kurnatovskiy yes, you're right. We updating username right after success sign up.